jenkinsfiles/Jenkinsfile.create-builder-template

```diff
@@ -0,0 +1,49 @@
+withCredentials([usernamePassword(credentialsId: 'e9f31c41-6d0e-4950-b6d7-1aa7080468aa', passwordVariable: 'VC_PASSWORD', usernameVariable: 'VC_USERNAME')]) {
```

Comment:
Consider reworking this to follow *declarative pipeline* style from `Jenkinsfile.deploy-builder` in the old repository.
Please. Strongly consider.